ripe/atlas/tools/settings/__init__.py

```diff
@@ -241,4 +241,31 @@ def write(config):
             rc.write(payload)
 
 
+class Aliases(Configuration):
+    """
+    A singleton class to manage user aliases
+    """
+
```

Comment:
How is that class singleton? I don't see Configuration class to be singleton as well? @danielquinn any ideas? I think you were the initial author of this right? maybe wrong comment? or am I missing something?